test/unit/event.js

```diff
@@ -2686,6 +2686,134 @@ test( "Inline event result is returned (#13993)", function() {
+	} );
+	jQuery( "#donor-input" ).on( "click", function( event ) {
+		assert.ok( true, "got a click event from the input" );
+		assert.ok( !event.isPropagationStopped(), "propagation says it's not stopped" );
+		assert.equal( event.type, "click", "make sure event type is correct" );
+		assert.equal( typeof event.originalEvent, "object", "make sure originalEvent exist" );
+	} );
+	jQuery( "#donor-input" ).on( "change", function( event ) {
+		assert.equal( typeof event.originalEvent, "object", "make sure originalEvent exist" );
+		assert.equal( event.type, "change", "make sure event type is correct" );
+		assert.ok( true, "got a change event from the input" );
+		event.stopPropagation();
+	} );
+	jQuery( "#donor-input" )[0].click();
+} );
+
+test( "originalEvent property for IE8", function( assert ) {
+	if ( !(/msie 8\.0/i.test( window.navigator.userAgent )) ) {
+		assert.expect( 1 );
+		assert.ok( true, "Assertions should run only in IE" );
+		return;
+	}
+
+	assert.expect( 12 );
+
+	var html = "<div id='donor-outer'>" +
+		"<form id='donor-form'>" +
+			"<input id='donor-input' type='radio' />" +
+		"</form>" +
+	"</div>";
+
+	jQuery( "#qunit-fixture" ).append( html );
+
+	jQuery( "#donor-outer" ).on( "change", function( event ) {
+		assert.ok( true, "click bubbled to outer div" );
+		assert.equal( event.originalEvent.type, "click", "make sure simulated event is a click" );
+		assert.equal( event.type, "change", "make sure event type is correct" );
+	} );
+
+	jQuery( "#donor-outer" ).on( "click", function( event ) {
+		assert.ok( true, "click bubbled to outer div" );
+		assert.equal( event.originalEvent.type, "click", "make sure originalEvent exist" );
+	} );
+	jQuery( "#donor-input" ).on( "click", function( event ) {
+		assert.ok( true, "got a click event from the input" );
+		assert.ok( !event.isPropagationStopped(), "propagation says it's not stopped" );
+		assert.equal( event.originalEvent.type, "click", "make sure originalEvent exist" );
+		assert.equal( event.type, "click", "make sure event type is correct" );
+	} );
+	jQuery( "#donor-input" ).on( "change", function( event ) {
+		assert.equal( event.originalEvent.type, "click", "make sure originalEvent exist" );
+		assert.equal( event.type, "change", "make sure event type is correct" );
+		assert.ok( true, "got a change event from the input" );
+	} );
+	jQuery( "#donor-input" )[0].click();
+} );
+
+test( "originalEvent property for Chrome, Safari and FF of simualted event", function( assert ) {
+	var userAgent = window.navigator.userAgent;
+
+	if ( !(/chrome/i.test( userAgent ) ||
+	       /firefox/i.test( userAgent ) ||
+	       /safari/i.test( userAgent ) ) ) {
+		assert.expect( 1 );
+		assert.ok( true, "Assertions should run only in Chrome, Safari and FF" );
+		return;
+	}
+
+	assert.expect( 4 );
+
+	var html = "<div id='donor-outer'>" +
+		"<form id='donor-form'>" +
+			"<input id='donor-input' type='radio' />" +
+		"</form>" +
+	"</div>";
```

Comment:
For improved atomicity and encapsulation.

1. If this test would become obsolete and you would need to remove it you might not remove the fixtures you might forget to remove them too, this happened couple times before.

2. If there would be different assertion like `equal(  $( "div" ).length, 2 )` you would break it

3. Other tests might start use those fixtures too and then when you would need to change it, for regression testing or such, you might break other tests.

And stuff like that